src/components/button/examples/button-reduce-presence.tsx

```diff
@@ -0,0 +1,97 @@
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * Reduce Presence
+ *
+ * This example is identical to the one above, except that here, the
+ * `reducePresence` property has been set to `true`. This will hide the button
+ * when it is disabled. However, it will also make sure that the button remains
+ * visible while the loading animation is ongoing. When the animation is done
+ * and the checkmark has been shown, the button will hide.
+ *
+ * ###### Usage of `reducePresence` property
```

Comment:
It's never great to be honest. We should just remove it. 

I also wonder another thing. First line in these "comments" in the docs always becomes a `h5` that's why we have to write other headings under it as `h6`. @jgroth Can we set the first line to be a normal `p` instead, or at least set it to be `h2`? `h5` is brutal, and makes writing a good documentation really hard.

Reply:
Yeah that should not be hard to fix I think, I'm not sure how it become like this with `h5` :smile:

Reply:
I think I know, kind of 🙂 

Take this example:

![image](https://user-images.githubusercontent.com/154725/99082879-0b1a7380-25c5-11eb-868a-8208df612967.png)

- Button group is h1
- Usage and Layout are h2
- Examples is h3
- Icon only is h5

So for one, there's a gap between the h3 of *Examples* and the h5 of each example's heading. But also, *Examples* shouldn't be h3, because it's not a subsection of *Layout*.

So *Examples* should be promoted to an h2, and each example's first line should be promoted to h3 👍

Reply:
I'm resolving this thread (the one about heading levels in the docs), since it's something we'll have to fix in Kompendium.